<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_import</class>
 <widget class="QDialog" name="Dialog_import">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загруженный файл</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_created_utc" stretch="1,6">
       <item>
        <widget class="QLabel" name="label_created_utc">
         <property name="text">
          <string>Дата/время загрузки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_created_utc">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_reg_num" stretch="1,6">
       <item>
        <widget class="QLabel" name="label_reg_num">
         <property name="text">
          <string>Регистрационный номер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_reg_num">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_subject" stretch="1,6">
       <item>
        <widget class="QLabel" name="label_subject">
         <property name="text">
          <string>Субъект войск</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_subject">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_contacts" stretch="1,6">
       <item>
        <widget class="QLabel" name="label_contacts">
         <property name="text">
          <string>Контакты отправителя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_contacts">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_files" stretch="1,2,2,2">
       <item>
        <widget class="QLabel" name="label_files">
         <property name="text">
          <string>Файлы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_received_file">
         <property name="text">
          <string>Поступивший файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_protocol_import">
         <property name="text">
          <string>Протокол импорта</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_protocol_identify">
         <property name="text">
          <string>Протокол идентификации</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_stat" stretch="1,6">
       <item>
        <widget class="QLabel" name="label_stat">
         <property name="text">
          <string>Статистика</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_stat">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_status" stretch="1,6">
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>Статус работы с файлом</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_status">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_rows">
       <item>
        <widget class="QTableView" name="tableView_rows"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_operations" stretch="1,2,2,2">
       <item>
        <widget class="QLabel" name="label_operations">
         <property name="text">
          <string>Операции</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_import_delete">
         <property name="text">
          <string>Удалить загрузку</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_result_file">
         <property name="text">
          <string>Результирующий файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_import_finish">
         <property name="text">
          <string>Завершить работу</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
